pkg/defaults/policies/files/red_hat_images_signed.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "c6fd7cb3-de34-4918-be4c-370006330357",
+  "name": "Red Hat Images must be signed by the Red Hat Release Key",
+  "description": "Alert when Red Hat images are not digitally signed by Red Hat",
+  "rationale": "Red Hat Images should be signed using Red Hat's Product Signing Keys (https://access.redhat.com/security/team/key)",
+  "categories": [
+    "Supply Chain Security"
+  ],
+  "lifecycleStages": [
+    "DEPLOY"
+  ],
+  "severity": "HIGH_SEVERITY",
+  "lastUpdated": "2025-06-12T11:36:56.396127178Z",
```

Comment:
do we need a last updated field since when policy instance is first created? this can be removed

Reply:
Thanks! I based this policy definition on an export of a manually created policy and forgot to remove this. Done in https://github.com/stackrox/stackrox/pull/15763/commits/fa552c22c2132fe8f31ff775d096939f08f25a3f